sdk/core/src/types.rs

```diff
@@ -224,3 +224,47 @@ impl Tx {
         self.data = data;
     }
 }
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+#[wasm_bindgen]
+pub struct Commitment {
+    commitment: String,
+    secret_blinding_factor: String,
+}
```

Comment:
@ChrisCho-H The struct is used because of `wasm_bindgen` return type?
I'm wondering is it okay commit and secret is in the same object with public